dandiapi/api/views/upload.py

```diff
@@ -80,14 +80,15 @@ class UploadCompletionResponseSerializer(serializers.Serializer):
 def blob_read_view(request: Request) -> HttpResponseBase:
     request_serializer = DigestSerializer(data=request.data)
     request_serializer.is_valid(raise_exception=True)
-    if request_serializer.validated_data['algorithm'] not in supported_digests:
```

Comment:
per discussion in #483 let's support both `GET` and for just awhile "good old" `POST`. Depending on how that to be implemented "the best djangowy way" (@dchiquito ?) , `operation_summary` for `POST` can state that it is `DEPRECATED! Use GET instead`

Reply:
I just tried to implement it and it turns out that GET requests do not have request bodies, which is what is being used here to send in the algorithm and value. Updating to use query parameters would be outside the scope of this PR, so I vote we file an issue and handle it later.

Reply:
Isn't for GET it should just be provided in  request header?